planet/Decoration.ocd/Village.ocd/Lantern.ocd/Script.c

```diff
@@ -12,9 +12,13 @@ protected func Initialize()
 	return SetAction("Shine");
 }
 
+public func Definition(proplist def) {
```

Comment:
``Could you please write the curly braces in the next line, as it is the case for all the other functions?

Reply:
Will do.